browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchResult.java

```diff
@@ -0,0 +1,28 @@
+package com.braintreepayments.browserswitch;
+
+public class BrowserSwitchResult {
+
+    final private int status;
+    final private String errorMessage;
+
+    public static final int STATUS_OK = 1;
+    public static final int STATUS_CANCELED = 2;
+    public static final int STATUS_ERROR = 3;
```

Comment:
Would it make sense to wrap these 3 values in an `enum` type?

Reply:
They were enums originally I believe, but Enums on Android kind of have a [bad rap](https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3). During the refactoring seemed like a good opportunity to remove them.

Reply:
I'm good with not using an `enum` if it negatively impacts performance. My main concern was the lack of value checking since the constructors accept plain `int` values. Have we considered using the Android `Typedef` annotations as an `enum` replacement?

https://developer.android.com/studio/write/annotations#enum-annotations

Reply:
Yeah, this is a legitimate concern. I tried the enum annotations out once before and ran into some weirdness with mockito. I'll give it another shot to see if I can get them to work.